### DISCUSSION TITLE
The styling for selecting a month in datepicker is still native

### DISCUSSION THREAD

**Discussion by haifan420** [filamentphp/filament#9106] [Help] [answered]
Here is a screenshot:
![Screenshot_20231015_100232](https://github.com/filamentphp/filament/assets/117128195/641749c0-a9f8-4d7d-9e54-b4f04a575750)

```
Forms\Components\Datepicker::make('date_of_birth')
                            ->displayFormat('d/m/Y')
                            ->native(false)
                            ->required(),
```

**Comment by leandrocfe** [comment]
What is the Filament version?
![image](https://github.com/filamentphp/filament/assets/3833889/c3ba10aa-c266-4f26-a472-957ca8e545c5)
![image](https://github.com/filamentphp/filament/assets/3833889/60d8c6b2-d8b7-4b59-93c8-07a6fb0ddda4)

**Reply by haifan420** [reply]
Sorry for not specifying the version, I'm using v3.0.0.0.
However, your screenshot looks like a native HTML5 styling to me, because it doesn't look 'filament'-ish, if you know what i mean. It seems the native(false) is not applied to the month selection in datepicker

Here is a normal select without native(false). I think mine looks like this because that is just how native HTML5 styling in my browser
![image](https://github.com/filamentphp/filament/assets/117128195/96063c32-a120-436e-8b3a-646f9b6e3df0)
with native(false)
![Screenshot_20231015_195234](https://github.com/filamentphp/filament/assets/117128195/b5b1dea3-07d9-4b52-9423-ff473bd78e97)

FYI I'm using chrome, and this is what it looks like on ms edge
![image](https://github.com/filamentphp/filament/assets/117128195/b1062873-d6b7-4669-acca-0d9e57b2c3e0)

**Reply by leandrocfe** [reply, marked as answer]
You should update the packages
`composer update -W`

Check your composer.json and make sure your Filament package has ^3.0

**Reply by roncarino** [reply]
Same to me. I am already at ^3.0.

![Screenshot 2025-06-20 093204](https://github.com/user-attachments/assets/79939835-ac60-4279-8230-3f4c763e2a53)
